src/importexport/imagesexport/internal/pngwriter.cpp

```diff
@@ -69,7 +69,7 @@ mu::Ret PngWriter::write(INotationPtr notation, Device& destinationDevice, const
     RectF pageRect = page->abbox();
 
     if (TRIM_MARGIN_SIZE >= 0) {
-        QMarginsF margins(TRIM_MARGIN_SIZE, TRIM_MARGIN_SIZE, TRIM_MARGIN_SIZE, TRIM_MARGIN_SIZE);
+        QMarginsF margins(TRIM_MARGIN_SIZE + 230, TRIM_MARGIN_SIZE + 420, TRIM_MARGIN_SIZE + 220, TRIM_MARGIN_SIZE + 340);
```

Comment:
These hardcoded margins look odd. They may cause some problems in the feature because this code will be used for MuseScore.com. Please, remove them and I'll merge your PR after that. It is better to create a separate issue for the margins problem and solve it in a separate PR